analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/HashesShouldHaveUnpredictableSaltBase.cs

```diff
@@ -70,57 +73,11 @@ private void ProcessObjectCreation(ProgramState state, IObjectCreationOperationW
         }
     }
 
-    private static ProgramState ProcessInvocation(ProgramState state, IInvocationOperationWrapper invocation)
+    protected override ProgramState ProcessInvocation(ProgramState state, IInvocationOperationWrapper invocation)
     {
-        if (IsCryptographicallyStrongRandomNumberGenerator(invocation)
-            && FindMethodArgument(state, invocation, KnownType.System_Byte_Array) is { } dataArgument
-            && dataArgument.TrackedSymbol() is { } trackedSymbol)
-        {
-            return state.SetSymbolConstraint(trackedSymbol, ByteCollectionConstraint.CryptographicallyStrong);
-        }
-        else if (invocation.TargetMethod.Is(KnownType.System_Text_Encoding, nameof(Encoding.GetBytes))
-                 && FindMethodArgument(state, invocation, KnownType.System_String)?.AsLiteral() is { })
-        {
-            return state.SetOperationConstraint(invocation, ByteCollectionConstraint.CryptographicallyWeak);
-        }
-        else
-        {
-            return state;
-        }
+        state = base.ProcessInvocation(state, invocation) ?? state;
```

Comment:
Not needed anymore (duplicate comment)
```suggestion
        state = base.ProcessInvocation(state, invocation);
```